src/client/components/app-core/text.css

```diff
@@ -2,7 +2,7 @@
 
 body {
   font-family: le-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-  font-size: var(--font-size-default);
+  font-size: 112.5%; /* 18px */
```

Comment:
why change the default font size? Also if 112.5% equals 18px wouldn't it be better if you just change the font-size-default to 18px?